src/mavlink_server.cpp

```diff
@@ -41,8 +41,9 @@ MavlinkServer::MavlinkServer(const ConfFile &conf, std::vector<std::unique_ptr<S
     , _is_running(false)
     , _timeout_handler(0)
     , _broadcast_addr{}
-    , _system_id(-1)
-    , _comp_id(MAV_COMP_ID_CAMERA2)
+    , _is_sys_id_found(false)
+    , _system_id(1)
+    , _comp_id(MAV_COMP_ID_CAMERA)
```

Comment:
Do you initialize this in cases with or without Gazebo ?

Reply:
This is an usused variable, need to be removed later